conf/st2.conf.sample

```diff
@@ -6,17 +6,21 @@
 enable = True
 
```

Comment:
Can you add a commented config section in other conf files as well? We've been repeatedly been asked about it in support.